src/components/ui/SlideOver/SlideOver.tsx

```diff
@@ -56,6 +56,10 @@ const SlideOver = ({
         layout.current.classList.add('no-scroll')
       }
     }
+
+    return () => {
+      layout.current?.classList?.remove('no-scroll')
```

Comment:
Now I'm not happy about where I placed this... This component has 3 `useEffect`s, but adding this to the 1st one doesn't fix the issue, while it works for the 2nd and 3rd ones. Thinking where the is best logical place to place this.
🤔 

<img width="559" alt="CleanShot 2022-02-16 at 20 30 46@2x" src="https://user-images.githubusercontent.com/381395/154375405-cfebd020-d325-4639-afc8-b4cbb3fbc8a9.png">

Reply:
Hmm. Actually, I think it's okay to use multiple effects here. 🤔 

As suggested by the documentation: [Use Multiple Effects to Separate Concerns](https://reactjs.org/docs/hooks-effect.html#tip-use-multiple-effects-to-separate-concerns)

Reply:
Btw, I don't think to verify `handleClose` (line 66). Can you check it again and then remove it pls ✨

Reply:
I also think it's okay to leave on that `useEffect` as it's related to the `layout`.

I believe it didn't work for the first effect because when the cleanup function is called the component is already on unmount state, possibly without the current `layout` value, or possibly it has changed. You can try to store the `layout.current` value inside the `useEffect` scope to keep the value (see this discussion [here](https://stackoverflow.com/questions/67069827/cleanup-ref-issues-in-react))